### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by aaranadev**
I’ve seen during the past that bugs and feature requests are not following a common format. I’d like to launch a proposal using github templates to standardize the way we recover bugs or feature requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (alasarr) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: I miss here the environment and logs.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (alasarr) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Just description

**Review by alasarr — CHANGES_REQUESTED** [review]
I feel issue forms would fit better

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aaranadev) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: Ok, no problem. The .md format is defined by the templates edit form of GitHub

Screenshot 2022-10-07 at 8 49 47 AM

I'll change it to `yml`

**Pull request closed without merging**